libwget/ssl_wolfssl.c

```diff
@@ -594,10 +594,9 @@ void wget_ssl_init(void)
 
 		debug_printf("WolfSSL init\n");
 		wolfSSL_Init();
+#ifndef  NO_OLD_TLS
```

Comment:
The `NO_OLD_TLS` seems to be something from your build system!?
Can we just use TLSV1_2 or higher as default for "auto" and still allow users to explicitly specify older versions? Because there are cases where ancient embedded systems only understand older protocol versions. It would be tedious if you need to build your own version of wget2 just to connect to those systems.

Reply:
You would think but no that isn't my declare:) Wolfcrypt is a bit odd and and a massive slew of their own declares around:


NO_OLD_TLS is theirs.   If it isn't defined you won't be getting TLSV1_1 even if you want it:).

Reply:
Thanks for the link. I wish there was a way to figure out the built-in capabilities at runtime. But here we go 🤷🏻